### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by polshinsm**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -88,4 +88,14 @@ RUN rm /var/cache/apk/* \
 COPY config /root/.kube/config
 COPY settings.xml /root/.m2/settings.xml
 
-CMD bash
\ No newline at end of file
+CMD bash
+
+FROM node:alpine
```
Comment (sta-szek) [inline Dockerfile]: docker must be light, please do not extend from any others, just add required dependency

File: Dockerfile [hunk 2]
```diff
@@ -88,4 +88,14 @@ RUN rm /var/cache/apk/* \
...
+
+RUN npm install -g redoc-cli
+
+WORKDIR /data
```
Comment (sta-szek) [inline Dockerfile]: remove

File: Dockerfile [hunk 3]
```diff
@@ -88,4 +88,14 @@ RUN rm /var/cache/apk/* \
...
+EXPOSE 8080
```
Comment (sta-szek) [inline Dockerfile]: why

File: Dockerfile [hunk 4]
```diff
@@ -88,4 +88,14 @@ RUN rm /var/cache/apk/* \
...
+
+ENTRYPOINT ["redoc-cli"]
```
Comment (sta-szek) [inline Dockerfile]: should be `CMD bash`

File: Dockerfile [hunk 5]
```diff
@@ -88,4 +88,14 @@ RUN rm /var/cache/apk/* \
...
+CMD []
```
Comment (sta-szek) [inline Dockerfile]: should be `CMD bash`

**Review by sta-szek — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 3 above)
Comment (polshinsm) [reply Dockerfile]: It's can be started as server to process incoming requests to this port

File: Dockerfile (hunk 1 above)
Comment (polshinsm) [reply Dockerfile]: +

File: Dockerfile (hunk 2 above)
Comment (polshinsm) [reply Dockerfile]: +

File: Dockerfile (hunk 3 above)
Comment (polshinsm) [reply Dockerfile]: I removed it too. i think we will use just html bundle

File: Dockerfile (hunk 4 above)
Comment (polshinsm) [reply Dockerfile]: +

**Review by sta-szek — APPROVED** [review]

**Pull request merged**